package.json

```diff
@@ -39,8 +39,8 @@
     "npm": ">= 6"
   },
   "dependencies": {
-    "@webdoc/model": "1.1.3",
-    "@webdoc/template-library": "1.1.3",
+    "@webdoc/model": "1.2.1",
+    "@webdoc/template-library": "1.2.1",
```

Comment:
Can we convert these to peerDependencies instead? And maybe do a fuzzy version of `^1.2.1`?

Reply:
@bigtimebuddy , sure, updated